src/lucky/redirectable.cr

```diff
@@ -60,14 +60,35 @@ module Lucky::Redirectable
   # ```
   # Note: It's recommended to use the method above that accepts a human friendly version of the status
   def redirect(to path : String, status : Int32 = 302) : Lucky::TextResponse
-    context.response.headers.add "Location", path
-    context.response.headers.add "Turbolinks-Location", path
-    context.response.status_code = status
-    Lucky::TextResponse.new(context, "", "")
+    if request.headers["X-Requested-With"]?.try(&.downcase) == "xmlhttprequest" && request.method != "GET"
```

Comment:
We also do this same line here https://github.com/luckyframework/lucky/blob/master/src/lucky/mime_type.cr#L106. Maybe this can be used somehow? Or play back in to this thing https://github.com/luckyframework/lucky/blob/master/src/lucky/request_type_helpers.cr

Reply:
first method is private

about second, it not works for me https://github.com/luckyframework/lucky/issues/1131

Reply:
@skojin After this would you mind opening another PR that uses this logic for the `ajax?` check? If not that's ok, just wondering :)

Reply:
yes, sure

Reply:
Thank you!